packages/itwinui-css/src/input/input.scss

```diff
@@ -81,3 +81,12 @@
     resize: none;
   }
 }
+
+@mixin iui-search-box {
```

Comment:
this mixin should probably be renamed. it gives the false impression that it contains the code for the whole searchbox component